lib/ClimaCoreVTK/src/ClimaCoreVTK.jl

```diff
@@ -127,5 +127,40 @@ function writevtk(
     end
 end
 
+"""
+    writevtk(
+        basename::String,
+        times,
+        fields,
+        [ispace=vtk_space(first(first(fields)));
+        vtkargs...
+    )
+
+Write a sequence of NamedTuple fields `fields` at times `times` as a Paraview collection.
+"""
+function writevtk(
+    basename::String,
+    times,
+    fields::NamedTuple,
+    ispace = vtk_space(first(first(fields)));
+    vtkargs...,
+)
+    npad = ndigits(length(times); pad = 3)
+
+    for (key, val) in pairs(fields)
+        paraview_collection(basename) do pvd
+            for (n, time) in enumerate(times)
+                field = NamedTuple{(key,)}([val[n]])
+                pvd[time] = vtk_file(
+                    basename * "_" * string(n, pad = npad),
+                    field,
+                    ispace;
+                    vtkargs...,
+                )
+            end
+        end
+    end
```

Comment:
I don't think you want the `for` loop on the outside (that will create multiple collections, but they will overwrite each other as they have the same name). Do you want something like this:

```suggestion
        paraview_collection(basename) do pvd
            for (n, time) in enumerate(times)
                field = NamedTuple(key => val[n] for (key, val) in pairs(fields))
                pvd[time] = vtk_file(
                    basename * "_" * string(n, pad = npad),
                    field,
                    ispace;
                    vtkargs...,
                )
            end
        end
```
?

Reply:
Doh! I had tried the loop in the inside, but for some reason when I was testing this I thought I had gotten the expected behavior. Thanks so much!